azureiot.sln

```diff
@@ -12,6 +12,9 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "security", "security", "{96531499-C8F4-4F77-88DD-9F9585B5E565}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "e2e", "e2e", "{662CE01D-7A7F-48D8-80CE-5DC9C60677E8}"
+	ProjectSection(SolutionItems) = preProject
+		iothub\device\src\DeviceAuthenticationWithRegistrySymmetricKey.cs = iothub\device\src\DeviceAuthenticationWithRegistrySymmetricKey.cs
+	EndProjectSection
```

Comment:
this looks like a testing change that went in accidentally

Reply:
Oh I commented saying it might have come in accidentally and has been fixed since.